src/cc/tools/qfsfileenum_main.cc

```diff
@@ -72,16 +75,20 @@ main(int argc, char **argv)
             case 'c':
                 config = optarg;
                 break;
+            case 'e':
+                showErrorsOnlyFlag = true;
+                break;
             default:
                 help = true;
                 break;
         }
     }
 
-    if (help || ! server || port < 0 || ! filename) {
+    if (help || ! server || port < 0) {
         cout << "Usage: " << argv[0] <<
-            " -s <server name> -p <port> -f <path> [-v] -c <config file>\n"
-            "Enumerate the chunks and sizes of the given file.\n";
+            " -s <server name> -p <port> [-f <path>] [-v] -c <config file>"
```

Comment:
I have created https://quantcast.atlassian.net/browse/QFS-289 to address this help later.